src/models/requestType.js

```diff
@@ -19,6 +19,16 @@ class RequestType extends Model {
       },
     };
   }
+
+  $afterValidate(json) {
+    super.$afterValidate(json);
+    const requests = json;
+    if (requests.type === '') {
```

Comment:
Let's make this `requests.trim().length === 0` otherwise this condition won't catch it when requests is `  ` (double space)